public/js/actions/breakpoints.js

```diff
@@ -24,8 +32,16 @@ function _getOrCreateBreakpoint(state, location, condition) {
   return getBreakpoint(state, location) || fromJS({ location, condition });
 }
 
-function addBreakpoint(location, { condition, getTextForLine } = {}) {
-  return ({ dispatch, getState, client }) => {
+type AddBreakpointOpts = {
+  condition?: string,
+  getTextForLine?: (line: number) => string
+};
+
+function addBreakpoint(
+  location: Location,
+  { condition, getTextForLine } : AddBreakpointOpts = {}
```

Comment:
It definitely does. I just wanted to get this PR out. It's probably because I'm new to flow that this is verbose. We'll figure out the right balance over time, and I'll look at this again before merging.